pkgs/top-level/python-packages.nix

```diff
@@ -12771,4 +12771,19 @@ let
     };
   };
 
+  pychart = buildPythonPackage rec {
+    name = "pychart-1.39";
+
+    src = pkgs.fetchurl {
+      url = "http://download.gna.org/pychart/PyChart-1.39.tar.gz";
+      md5 = "f1f509a1c4623056c8e780bb7c9a05c5";
+    };
+
+    meta = with stdenv.lib; {
+      description = "PyChart is a Python library for creating high quality Encapsulated Postscript, PDF, PNG, or SVG charts.";
```

Comment:
The package name should not be containted in the description. See http://nixos.org/nixpkgs/manual/#idm140737319048304